src/status_im/utils/pre_receiver.cljs

```diff
@@ -39,6 +36,6 @@
             (do (swap! seen disj [clock-value id])
                 (add-fn msg))
             (async/put! mature-ch msg))
-          (add-fn msg))
-        (recur)))
+          (add-fn msg)))
+      (recur))
```

Comment:
I was very quite confused by this and for the moment thought that the `(recur)` is scoped in the outermost `if`, that's why this move, so it's clearly one level under `go-loop`

Reply:
Hm, IMO it is good form to always have a `let` be a single form for parent expression, but guess this is fine too.